libs/akita-schematics/src/ng-g/utils/parse-name.ts

```diff
@@ -6,13 +6,13 @@ export interface Location {
 }
 
 export function parseName(options: any): Location {
-  const { name, path, dirName, feature } = options;
+  const { name, path, dirName, feature, flat } = options;
   const nameWithoutPath = basename(name as Path);
   const namePath = dirname((path + '/' + name) as Path);
-  const normalizedPath = feature ? normalize('/' + namePath + '/' + dirName) : normalize('/' + namePath);
+  const normalizedPath = feature ? normalize('/' + namePath + '/' + dirName + (flat ? '' : '/' + nameWithoutPath)) : normalize('/' + namePath);
```

Comment:
Can you refactor it please to make it more readable? Maybe separate it into variables and use template strings.

Reply:
sure, will do it tomorrow

Reply:
done